components/Mailer/basic_usage.rst

```diff
@@ -0,0 +1,193 @@
+Basic usage
+===========
+
+.. _component_mailer_provider_default-settings-provider:
+
+DefaultSettingsProvider
+-----------------------
```

Comment:
This is not that important service and usage example. First should be Sender and sending e-mails, don't you think? :) I know that in order to construct sender and actually send out the e-mail you need to use these, but I think it is more natural to see what is the main use case for the bundle and then go deeper into what each element does etc. Thoughts?